src/style/list/conversations.less

```diff
@@ -78,22 +93,26 @@
     &.active {
       background-color: var(--gray-95);
     }
-    .conversations-footer-btn--text {
+    .conversations-sidebar-btn--text {
       color: var(--gray-50);
     }
   }
```

Comment:
the whole `body.theme-dark` can be removed here
white text on accent-color background is identical for light and dark mode for current active conversation